kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/build/BuildMojoTest.java

```diff
@@ -52,7 +53,7 @@ void buildServiceConfigBuilder_shouldReturnNonNullResourceConfigIfConfigured() {
         BuildServiceConfig.BuildServiceConfigBuilder buildServiceConfigBuilder = buildMojo.buildServiceConfigBuilder();
         // Then
         assertThat(buildServiceConfigBuilder.build()).isNotNull()
-            .returns("src/main/jkube", c -> c.getResourceDir().getPath())
+            .returns(separatorsToSystem("src/main/jkube"), c -> c.getResourceDir().getPath())
```

Comment:
This is okay, but why don't rely on using simple `File` to do the conversion automatically for us?

In previous cases, we had to use `separatorsToSystem` as FileAssertions required comparing file paths as string literals. We might want to refactor these classes in the future to do the path conversion automatically.

Reply:
Oh yeah, You are right, I didn't realize, here, we can use `File` for conversion. I'll pay more attention to such detail from now on. Thanks for the review. 

I'll push a commit with a recommended changed soon.

Reply:
File or Path

Reply:
I guess we can use one of the two, I'm going to use `File` for now.

Reply:
I've made the recommended changes. I've used `File` instead of `Path` to assert the return values as the actual return value of `getResourceDir` is `FIle` object only.